pkg/apis/kops/validation/legacy.go

```diff
@@ -400,6 +400,34 @@ func ValidateCluster(c *kops.Cluster, strict bool) field.ErrorList {
 		}
 	}
 
+	if c.Spec.IAMRolesForServiceAccounts != nil {
```

Comment:
Could this go into `validateClusterSpec()` instead of legacy.go? Let's try not to add to tech debt.